### DISCUSSION TITLE
change date for an order

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3195] [Q&A]
Hello!

Suppose that there is an order placed and a ticket is issued.
Is there a way for an administrator to change the date of this order (upon request of course) ?
Of course there could be a cancellation, then a refund and finally a new order. But is there a way to just change the date without having to refund a place a new order again ?

Thank you in advance.

**Comment by raphaelm** [comment]
Yes, the date of a series can be changed through the "change products" button in the backend order view.

**Comment by alemao8** [comment]
Thanx Raphael.
I haven't noticed it..
Can this be changed from the PretixPOS mob app as well ?

**Reply by raphaelm** [reply]
no currently not
